python/tests/stress/arcticdb/version_store/test_stress_delete.py

```diff
@@ -32,34 +33,35 @@ def test_stress_delete(object_store_factory):
     num_tests = 100
     dataframe_size = 1000
 
+    syms = []
     written_dfs = []
 
     for x in range(num_tests):
         symbol = "symbol_{}".format(x)
+        syms.append(symbol)
         df = sample_dataframe(dataframe_size, x)
-        lib1.write(symbol, df)
-        lib2.write(symbol, df)
         written_dfs.append(df)
 
+    lib1.batch_write(syms, written_dfs)
+    lib2.batch_write(syms, written_dfs)
+
     start_time = datetime.now()
     lib1.version_store.clear()
     print("Delete took {}".format(datetime.now() - start_time))
 
     # Make sure that the symbols are deleted
-    for x in range(num_tests):
-        with pytest.raises(NoDataFoundException) as e:
-            lib1.read(f"symbol_{x}")
+    with pytest.raises(NoDataFoundException) as e:
+        lib1.batch_read(syms)
 
-    for x in range(num_tests):
-        symbol = "symbol_{}".format(x)
-        assert_frame_equal(lib2.read(symbol).data, written_dfs[x])
+    res = lib2.batch_read(syms)
+    for i, sym in enumerate(syms):
+        assert_frame_equal(res[sym].data, written_dfs[i])
 
     check_no_keys(lib1)
 
     lib2.version_store.clear()
     check_no_keys(lib2)
 
     # Make sure that the symbols are deleted
-    for x in range(num_tests):
-        with pytest.raises(NoDataFoundException) as e:
-            lib2.read(f"symbol_{x}")
+    with pytest.raises(NoDataFoundException) as e:
+        lib2.batch_read(syms)
```

Comment:
This check is less comprehensive than the one before (the new check asserts that at least blows up, the old checked they all did)